### PR TITLE
Add simapp to CI

### LINKED ISSUE #242: Start Cosmos SDK 0.39 compatible chain in CI
In order to test the [upcoming](https://github.com/CosmWasm/cosmjs/issues/237) @cosmjs/sdk39, we need a version of gaia running in the CI.

Examples scripts:
- Gaia 2.0 (current CosmosHub): https://github.com/iov-one/iov-core/tree/master/scripts/cosmoshub
- Wasmd (Gaia fork): https://github.com/CosmWasm/cosmjs/tree/master/scripts/wasmd

This includes 3 things

- [x] Find out which version of gaia is the right one (see e.g. https://github.com/cosmos/gaia/issues/425; there is also an ibc-alpha branch 🤷‍♂️)
- [ ] Get docker images
- [ ] Create start/stop scripts

### PR DESCRIPTION
**Pull request by willclarktech**
Closes #242
~Merge after #275~

### CODE REVIEW

File: scripts/simapp/env [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Choose from https://hub.docker.com/r/cosmwasm/simapp/tags
+REPOSITORY="cosmwasm/simapp"
+VERSION="latest"
+CONTAINER_NAME="simapp"
```
Comment (webmaster128) [inline scripts/simapp/env:4]: Can we use the one from the Cosmos SD team? https://github.com/cosmos/cosmos-sdk/pull/6517#issuecomment-652901250 There we also get pinned versioning

File: scripts/simapp/template/.simapp/config/config.toml [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+
+wal_file = "data/cs.wal/wal"
+
+timeout_propose = "3s"
+timeout_propose_delta = "500ms"
+timeout_prevote = "1s"
+timeout_prevote_delta = "500ms"
+timeout_precommit = "1s"
+timeout_precommit_delta = "500ms"
+timeout_commit = "5s"
```
Comment (webmaster128) [inline scripts/simapp/template/.simapp/config/config.toml:272]: For wasmd we reduced those values to get (much) faster tests: https://github.com/CosmWasm/cosmjs/blob/master/scripts/wasmd/template/.wasmd/config/config.toml#L265-L271 I think we should do the same here.

File: scripts/simapp/env (hunk 1 above)
Comment (willclarktech) [reply scripts/simapp/env:4]: `interchainio/cosmos-sdk` is simapp?

Comment (webmaster128) [reply scripts/simapp/env:4]: It should be, yes

**Comment by webmaster128** [comment]
Arg

> Bind for 0.0.0.0:26657 failed: port is already allocated.

**Pull request merged**